### PR TITLE
Re-Sprite ERT Gear + New RIG Sound Effects + ERT Hardsuit Roles.

### PR DESCRIPTION
**Pull request by Dark1Star3s**
Basically, the entire goal of my mammoth PR is to Re-Sprite the ERT and Deathsquad suits, and give a bit of love to the current RIG/Hardsuits we have in game, by giving them sound effects based on certain things, power, errors, etc.
---
Sprites
- [X] Re-Sprite HAPT Hardsuit (Heavy Asset Protection AKA The Guys you send in to kill antags.)
- [X] Re-Sprite ERT-L Hardsuit (Emergency Response Team-Lead Trooper)
- [x] Re-Sprite ERT-S Hardsuit (Emergency Response Team-Security Trooper)
- [X] Re-Sprite ERT-E Hardsuit (Emergency Response Team-Engineering Trooper)
- [x] Re-Sprite ERT-M Hardsuit (Emergency Response Team-Medical Trooper)
https://i.gyazo.com/6123a0ef813da4ae6a5dc5c050867b0c.png new ERT sprites are completed, bar the Janitor. (but that's a meme) 

---
Sound Stuff

- [X] Add Powerdown/Failure Sounds.

- [X] Add Powerup Sounds.

- [X] Add Caution Alarm Sounds.

- [X] Add New Movement Sounds.

- [x] More stuff. Maybe.
---
Also added various changes to each ERT suit. Now each ERT suit has their own speciallity. 
* The Lead and Medical Hardsuit have standardised armor, they are the same, however one is obviously equipped with medical equipment.
* The Engineering ERT Hardsuit: is now immune against explosions, and has higher resistence to energy weaponry. This is the guy you throw at a bomb or engineering stuff.
* The Security ERT Hardsuit, has higher protection against Melee, and has better armor against ballistics. This is the guy you throw at antags.
* The HAPT ERT Hardsuit, has higher protection against Melee, Ballistics, Lasers, and slightly more protection against energy, it also has slight resistence to bombs.

The Descriptions of the ERT Hardsuits have been updated as well.
* The Lead ERT Hardsuit: ""A hardsuit, designed for response teams and the like. Designed to command the rest of the squad in the field. Has blue highlights."
* The Medical ERT Hardsuit: "A hardsuit, designed for response teams and the like. Designed for medical support in the field. Has white and red highlights."
* The Engineering ERT Hardsuit: "A hardsuit, designed for response teams and the like. Designed for Engineering use and Bomb Disposal. Has orange highlights."
* The Security ERT Hardsuit: "A hardsuit, designed for response teams and the like. Designed for front line security operations. Has red highlights."
* The HAPT ERT Hardsuit:  "A very heavy hardsuit, modified to take on the galaxies most hostile environments, this one appears to have a LOT of armour. Do not mess with the person wearing this."

### CODE REVIEW

**Comment by Roland410** [comment]
Heavy asset protection is not deathsquad afaik, but nice PR.

**Comment by BurgerLUA** [comment]
Make this PR into a draft if it's WIP, please.

**Comment by Tristan63** [comment]
i nut

**Comment by Roland410** [comment]
Please don't put descriptors which refer to armor values and such into the RIG descs.

**Comment by BurgerLUA** [comment]
This good to merge?

**Comment by Dark1Star3s** [comment]
Aye it is! She's ready.

File: code/modules/clothing/spacesuits/rig/suits/ert.dm [hunk 1]
```diff
@@ -50,9 +50,19 @@
 
 /obj/item/weapon/rig/ert/engineer
 	name = "emergency response engineering hardsuit control module"
-	desc = "A hardsuit used by many corporate and governmental emergency response forces. Has orange highlights. Armoured and space ready."
+	desc = "A hardsuit, designed for response teams and the like. Designed for Engineering use and Bomb Disposal. Has orange highlights."
```
Comment (Roland410) [inline code/modules/clothing/spacesuits/rig/suits/ert.dm:53]:
```suggestion
	desc = "A hardsuit, designed for response teams and the like. Designed for engineering use and bomb disposal. Has orange highlights."
```

**Pull request merged**